bpf/include/bpf_log.h

```diff
@@ -101,7 +101,7 @@ static inline int map_lookup_log_level()
     int *value = NULL;
     value = kmesh_map_lookup_elem(&bpf_log_level, &zero);
     if (!value)
-        return 0;
+        return BPF_LOG_INFO;
```

Comment:
This is a change, now no debug level log will be printed by default. Which can be tuned by user from kmesh-daemon.